solutions/usaco-921.mdx

```diff
@@ -79,9 +79,13 @@ int timer = 1;
 void dfs(int v=0,int p=-1)
```

Comment:
actually I think an easier way to fix this would be to change `p=-1` to `p=0`.

Reply:
um alright